packages/strapi-design-system/src/Grid/Grid.tsx

```diff
@@ -1,19 +1,23 @@
 import * as React from 'react';
 
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 import { GridContext } from './GridContext';
-import { Box } from '../Box';
+import { Box, BoxProps } from '../Box';
 import handleResponsiveValues from '../helpers/handleResponsiveValues';
 
-const GridWrapper = styled(Box)`
+export interface GridProps extends BoxProps<HTMLDivElement> {
```

Comment:
```suggestion
export interface GridProps extends BoxProps {
```

Iirc `HTMLDivElement` is the default value for this generic 👍🏼